File: app/src/androidTest/java/in/mvpstarter/sample/runner/TestRunner.java
package in.mvpstarter.sample.runner;

import android.app.Application;
import android.content.Context;

import in.mvpstarter.sample.MvpStarterApplication;
import io.appflate.restmock.android.RESTMockTestRunner;

/**
 * Created by ravindra on 4/2/17.
 */
public class TestRunner extends RESTMockTestRunner {

    @Override
    public Application newApplication(ClassLoader cl, String className, Context context)
            throws InstantiationException, IllegalAccessException, ClassNotFoundException {
        return super.newApplication(cl, MvpStarterApplication.class.getName(), context);
    }

}

File: app/src/androidTest/kotlin/in/mvpstarter/sample/runner/TestRunner.kt
package `in`.mvpstarter.sample.runner

import android.app.Application
import android.content.Context

import `in`.mvpstarter.sample.MvpStarterApplication
import io.appflate.restmock.android.RESTMockTestRunner

/**
 * Created by ravindra on 4/2/17.
 */
class TestRunner : RESTMockTestRunner() {

    @Throws(InstantiationException::class, IllegalAccessException::class, ClassNotFoundException::class)
    override fun newApplication(cl: ClassLoader, className: String, context: Context): Application {
        return super.newApplication(cl, MvpStarterApplication::class.java.name, context)
    }

}
